Pelias.podspec

```diff
@@ -12,13 +12,14 @@ Pod::Spec.new do |s|
   s.requires_arc = true
   s.ios.deployment_target = '9.3'
   s.osx.deployment_target = '10.11'
+  s.pod_target_xcconfig = { 'SWIFT_VERSION' => '3.0' }
 
   # Exclude optional Convenience modules
   s.default_subspec = 'Core'
 
   ### Subspecs
 
   s.subspec 'Core' do |cs|
-    cs.source_files   = 'src/*.swift'
```

Comment:
This change breaks compatibility with Swift Package Manager, which (until Swift 4) expects files to live in the `src/` directory.